snapper/Btrfs.cc

```diff
@@ -1446,19 +1453,19 @@ namespace snapper
     {
 	try
 	{
+	    SDir general_dir = openGeneralDir();
+
 	    if (num == 0)
 	    {
 		SDir subvolume_dir = openSubvolumeDir();
 		subvolid_t id = get_id(subvolume_dir.fd());
-		set_default_id(subvolume_dir.fd(), id);
+		set_default_id(general_dir.fd(), id);
```

Comment:
Just out of curiosity, why some methods like `set_default_id` are with underscores and other ones with camel case?